app/app.css

```
@@ -1,59 +1,64 @@
-/* Fonts */
-@import url('https://fonts.googleapis.com/css2?family=Instrument+Serif:ital@0;1&display=block');
-
-@import "tailwindcss";
+/* vite+ SSR currently fails to resolve package CSS imports during transform-only */
+@import "../node_modules/tailwindcss/index.css";
 
 @font-face {
-  font-family: 'Geist';
-  src: url('https://cdn.jsdelivr.net/npm/geist@1.2.0/dist/fonts/geist-sans/Geist-Regular.woff2') format('woff2');
+  font-family: "Geist";
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)